site-src/concepts/api-overview.md

```diff
@@ -1,12 +1,19 @@
 # API Overview
 
 ## Background
-The Gateway API Inference Extension project is an extension of the Kubernetes Gateway API for serving Generative AI models on Kubernetes. Gateway API Inference Extension facilitates standardization of APIs for Kubernetes cluster operators and developers running generative AI inference, while allowing flexibility for underlying gateway implementations (such as Envoy Proxy) to iterate on mechanisms for optimized serving of models.
-
-<img src="/images/inference-overview.svg" alt="Overview of API integration" class="center" width="1000" />
+Gateway API Inference Extension project optimizes self-hosting Generative AI Models on Kubernetes.
+This is achieved by leveraging Envoy's [External Processing](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter) to extend any gateway that supports both ext-proc and [Gateway API](https://github.com/kubernetes-sigs/gateway-api) into an [inference gateway](../index.md#concepts-and-definitions).
+This extension upgrades popular Gateways like Envoy Gateway, kGateway, and GKE Gateway - to become [Inference Gateways](../index.md#concepts-and-definitions) -
```

Comment:
s/kGateway/kgateway/

Consider using a different term than "upgrades"... maybe "converts"?

s/to become/into/

Reply:
@robscott should "inference gateway" be capitalized?

Reply:
I use `extends` instead of `upgrades`